src/commands/sbom/validation.ts

```diff
@@ -30,16 +32,30 @@ export const getValidator = (): Ajv => {
  * @param debug - if we need to show debug information
  */
 export const validateSbomFileAgainstSchema = (path: string, ajv: Ajv, debug: boolean): boolean => {
+  const showValidationErrors = (version: string, path: string, errors: ErrorObject[]): void => {
+    errors.forEach((message) => {
+      process.stderr.write(
+        `Error while validating file against CycloneDX ${version}: ${path}, ${message.schemaPath}: ${message.instancePath} ${message.message}\n`
+      )
```

Comment:
Could we pull out the `Error while validating file against CycloneDX ${version}: ${path}` from this loop, so we can have:

```
Error while validating file ${file} against CycloneDX ${version} schema:
  - ${message.schemaPath}: ${message.instancePath} ${message.message}\n
  - ${message.schemaPath}: ${message.instancePath} ${message.message}\n
  - ${message.schemaPath}: ${message.instancePath} ${message.message}\n
  - ${message.schemaPath}: ${message.instancePath} ${message.message}\n
```

instead of 

```
Error while validating file against CycloneDX ${version}: ${path}, ${message.schemaPath}: ${message.instancePath} ${message.message}\n
Error while validating file against CycloneDX ${version}: ${path}, ${message.schemaPath}: ${message.instancePath} ${message.message}\n
Error while validating file against CycloneDX ${version}: ${path}, ${message.schemaPath}: ${message.instancePath} ${message.message}\n
Error while validating file against CycloneDX ${version}: ${path}, ${message.schemaPath}: ${message.instancePath} ${message.message}\n
```